### PR TITLE
add import to API examples

### PR DESCRIPTION
**Pull request by xsrobin**

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -3414,6 +3433,7 @@ def layer_norm(input,
 
     Examples:
 
+            import paddle.fluid as fluid
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py:3436]: Please add ` >>> `

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -3495,6 +3515,7 @@ def group_norm(input,
...
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py:3518]: Please add `>>>`

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -5664,7 +5697,7 @@ def nce(input,
     Examples:
         .. code-block:: python
 
-
+            import paddle.fluid as fluid
 	    import numpy as np
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py:5701]: `import paddle.fluid as fluid` is not aligned with `import numpy as np`.

File: python/paddle/fluid/layers/nn.py [hunk 4]
```diff
@@ -8090,6 +8139,8 @@ def random_crop(x, shape, seed=None):
         ${out_comment}
 
     Examples:
+
+            import paddle.fluid as fluid
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py:8143]: Please add `>>>`

File: python/paddle/fluid/layers/ops.py [hunk 5]
```diff
@@ -135,6 +136,7 @@ def cumsum(x, axis=None, exclusive=None, reverse=None):
 cumsum.__doc__ = _cum_sum_.__doc__ + """
 Examples:
 
+        import paddle.fluid as fluid
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/ops.py:139]: Please add `>>>`

File: paddle/fluid/API.spec [hunk 6]
```diff
@@ -43,25 +43,24 @@ paddle.fluid.DataFeedDesc.set_use_slots (ArgSpec(args=['self', 'use_slots_name']
...
 paddle.fluid.CompiledProgram.with_inference_optimize (ArgSpec(args=['self', 'config'], varargs=None, keywords=None, defaults=None), ('document', '9e5b009d850191a010e859189c127fd8'))
-paddle.fluid.ExecutionStrategy.__init__ __init__(self: paddle.fluid.core_avx.ParallelExecutor.ExecutionStrategy) -> None
-paddle.fluid.BuildStrategy.GradientScaleStrategy.__init__ __init__(self: paddle.fluid.core_avx.ParallelExecutor.BuildStrategy.GradientScaleStrategy, arg0: int) -> None
-paddle.fluid.BuildStrategy.ReduceStrategy.__init__ __init__(self: paddle.fluid.core_avx.ParallelExecutor.BuildStrategy.ReduceStrategy, arg0: int) -> None
-paddle.fluid.BuildStrategy.__init__ __init__(self: paddle.fluid.core_avx.ParallelExecutor.BuildStrategy) -> None
-paddle.fluid.gradients (ArgSpec(args=['targets', 'inputs', 'target_gradients', 'no_grad_set'], varargs=None, keywords=None, defaults=(None, None)), ('document', 'e2097e1e0ed84ae44951437bfe269a1b'))
```
Comment (chengduoZH) [inline paddle/fluid/API.spec:50]: The signature of `paddle.fluid.gradients` is not generated.

File: python/paddle/fluid/framework.py [hunk 7]
```diff
@@ -157,7 +158,8 @@ def cpu_places(device_count=None):
     Examples:
         .. code-block:: python
 
-            cpu_places = fluid.cpu_places()
+        import paddle.fluid as fluid   
+        cpu_places = fluid.cpu_places()
```
Comment (haowang101779990) [inline python/paddle/fluid/framework.py:162]: 请保持原始缩进

**Pull request closed without merging**